src/Logger.php

```diff
@@ -185,9 +183,8 @@ public function log($level, $message, array $context = [])
             if ($this->traceLevel > 0) {
                 $count = 0;
                 $ts = debug_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS);
-                array_pop($ts); // remove the last trace since it would be the entry script, not very useful
                 foreach ($ts as $trace) {
-                    if (isset($trace['file'], $trace['line']) && strpos($trace['file'], YII_PATH) !== 0) {
```

Comment:
Instead of removing it, path could be made configurable.

Reply:
Same reasoning about this lib being usable outside of Yii.

Also, why wouldn't we trace files outside YII_PATH in the first place?

Reply:
That's not tracing for inside of YII_PATH, not outside. That's not to clutter logs.

Reply:
I was incorrectly reading the `!== 0`

`private $excludedTracesPaths = []` & a setter ?

Reply:
Implemented setExcludedTracePaths()